WordPressAuthenticator.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
-  s.version       = "1.10.0"
+  s.version       = "1.10.1"
```

Comment:
This should probably be a beta version, i.e. `1.10.1-beta.1`.

Reply:
Thanks, good catch!